pkg/collector/corechecks/servicediscovery/module/config_test.go

```diff
@@ -0,0 +1,95 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2024-present Datadog, Inc.
+
+//go:build test && linux_bpf
```

Comment:
The file has `_test.go` suffix
```suggestion
//go:build linux_bpf
```

Reply:
I removed test attribute